src/util/mod.rs

```diff
@@ -481,6 +485,49 @@ impl<T> Drop for MustConsumeVec<T> {
     }
 }
 
+pub fn make_cb(cmd: &RaftCmdRequest) -> (Callback, mpsc::Receiver<RaftCmdResponse>) {
```

Comment:
Why is this function placed in util?

Reply:
It belongs to `fn wait_cb` logically, it is also useful in other places like https://github.com/pingcap/tikv/pull/2625/files#diff-531835103966dedea15da38f6c49fdc5R239.

I choose here instead of `mod raftstore/util` because `wait_cb` knows `RaftStoreRouter`, a trait from `mod server`, and the raftstore should not import anything from the server.

Reply:
Bad design here. The util mod is lower than server and raftstore.